automation/repeated_test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+SCRIPT_PATH="$(
+    cd "$(dirname "${BASH_SOURCE[0]}")/" || exit
+    echo "$(pwd)/"
+)"
+
+NUM_TESTS=${NUM_TESTS:-3}
+if [[ -z ${NEW_TESTS} ]]; then
+    NEW_TESTS=$("${SCRIPT_PATH}"/new_tests.sh)
+    # skip certain tests for now, as we don't have a strategy currently
+    NEW_TESTS=$(echo "$NEW_TESTS" | sed -E 's/\|?[^\|]*(sriov|multus|windows|genie|gpu)[^\|]*//g')
+fi
+if [[ -z "${NEW_TESTS}" ]]; then
+    echo "Nothing to test"
+    exit 0
+fi
+
+TEST_LANES=( 'k8s-1.14.6' 'k8s-1.15.1' 'k8s-1.16.2' )
```

Comment:
Does it makes sense to have it configurable?

Reply:
I didn't make it configurable, thought about it at first, but don't see the need at the moment.

Reply:
Made it into an argument.